.travis.yml

```diff
@@ -14,7 +14,7 @@ jdk:
   - oraclejdk8
 
 script:
-  - ./gradlew clean build connectedCheck -x checkstyleTest --stacktrace
+  - ./gradlew clean build connectedCheck -x checkstyleTest -x :apollo-integration:transformClassesWithDexForDebug -x :apollo-integration:transformClassesWithDexForRelease --stacktrace
```

Comment:
`apollo-integration` module uses the maven repo version of the plugin which adds `apollo-api` and this conflicts with the one `apollo-runtime` adds (since this PR changed the scope to compile).